src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -105,6 +180,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     return recipesMatchingList;
   }
 
+  /**
+   * Returns a list of matched ingredients snippet by the search query.
+   */
+  private ArrayList<String> returnMatchedIngredients(ScoredDocument document) {
```

Comment:
returnMatchingIngredients

Always write the things in present with the exception when the past is related to the status / data.

In this case is just related to the fact that the query was executed before, but it's clearer to reflect it as "now".

Reply:
But the ingredients matching was made in the past, when the search is called. I only get the field from the ScoredDocument.